static/app/components/group/tagFacets/tagFacetsDistributionMeter.tsx

```diff
@@ -318,8 +317,18 @@ const LegendDot = styled('span')<{color: string; focus: boolean}>`
   border-radius: 50%;
   flex-shrink: 0;
   background-color: ${p => p.color};
-  transition: outline 0.3s;
-  ${p => (p.focus ? `outline: ${p.theme.gray100} ${space(0.5)} solid` : null)}
+  &:after {
+    content: '';
+    border-radius: 50%;
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    outline: ${p => p.theme.gray100} ${space(0.5)} solid;
+    opacity: ${p => (p.focus ? '1' : '0')};
+    transition: all 0.3s cubic-bezier(0.4, 0, 0.2, 1);
```

Comment:
```suggestion
    transition: opacity 0.3s cubic-bezier(0.4, 0, 0.2, 1);
```
small nit, if the know the only thing that's changing is opacity, we should probably just call that out in the transition